frontend/components/puzzle1.jsx

```diff
@@ -1,45 +1,56 @@
 import React from 'react';
+import styled from 'styled-components'
 
 // Answer: getaway (caesar ciphered)
 
+
+ //style component for incorrect answers
+   const AnswerAwareInput = styled.input`
```

Comment:
lets move this into Puzzle.jsx so we don't have to keep copy and pasting this

Reply:
This would require we pass the style down to the lower components somehow, right?

Reply:
We can actually define this in the Puzzle file and export it, then in here import it from Puzzle, that way we can reuse the same component in all the puzzles

Reply:
yup! although you'll probably need brackets since you're not exporting the default export, so `import { AnswerAwareInput } from './puzzle.jsx';`

Reply:
Sounds good. Made the change.